test/es6/module-syntax.js

```diff
@@ -0,0 +1,108 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// ES6 Module syntax tests -- verifies syntax of import and export statements
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "All valid (non-default) export statements",
+        body: function () {
+            assert.doesNotThrow(function () { WScript.LoadModuleFile('.\\module\\ValidExportStatements.js', 'samethread'); }, "Valid export statements");
+        }
+    },
+    {
+        name: "Valid default export statements",
+        body: function () {
+            assert.doesNotThrow(function () { WScript.LoadModuleFile('.\\module\\ValidExportDefaultStatement1.js', 'samethread'); }, "Unnamed function expression default export");
```

Comment:
I didn't do this for two reasons. First the test hook is experimental and might get replaced with a proper API. Second, different files makes it much easier to simulate real modules.

You're right, though. Figuring out what each of these files is testing is a pain. I went ahead and fleshed-out LoadModule in WScriptJsrt to load a module script from a buffer. The test cases are much more readable.